src/_premake_init.lua

```diff
@@ -2013,6 +2013,12 @@
 	filter { "system:darwin" }
 		toolset "clang"
 
+	filter "files:**.m"
+		compileas(p.OBJECTIVEC)
+
+	filter "files:**.mm"
+		compileas(p.OBJECTIVECPP)
```

Comment:
Is it possible to have these work the same as the other files instead of using `compileas`?